cargo-concordium/src/build.rs

```diff
@@ -1224,19 +1139,22 @@ pub(crate) fn build_and_run_integration_tests(
     if allow_debug {
         command.args(["--features", "concordium-std/debug"]);
     }
-    command.args(&cargo_args);
+    command.args(&cargo_extra_args);
     // when allowing debug output, we make sure that test output is not captured.
     if allow_debug {
         // check if the user has already supplied extra test flags
-        let (test_args, show_output) = cargo_args.iter().fold((false, false), |(ta, so), arg| {
-            if arg == "--" {
-                (true, so)
-            } else if arg == "--show-output" || arg == "--nocapture" {
-                (ta, true)
-            } else {
-                (ta, so)
-            }
-        });
+        let (test_args, show_output) =
```

Comment:
Rust formatter being annoying due to the longer `cargo_extra_args` rather than `cargo_args`